generator/templates/enums/template_numeric.h

```diff
@@ -0,0 +1,11 @@
+{% extends "template.h" %}
```

Comment:
Forgive me if I'm wrong, but `LightName` should be using the `name` not the value, right? This is generating numbers for that enum, but the current state is to generate strings.

Reply:
How to evaluate what enum should use `template_numeric`? In existing solution if there is value attribute, such enum evaluated as `template_numeric`. According to this logic `LightName` should be evaluated as `template_numeric`.
```python3
if any(map(lambda p: p.value, render['params'])):
    render['template'] = 'enums/template_numeric'
```
As well as part of refusing from any king of customisation, I suggest to delete `generator/templates/enums/template_numeric.h` and do not process string and number enums separately?

Reply:
I think we have to. I'm just surprised as both Java_Suite and iOS libs currently use the `name` value, not the `value` value. I'm talking with the Java_Suite team about what we should do.

Reply:
After discussion with the Core team and Java_Suite teams it should be okay as-is.

Reply:
After discussion with the Core team and Android team, we should remove all handling of "numeric" enums. It's okay that `PredefinedWindow` is handled incorrectly for the purposes of this generator.

Reply:
handling of "numeric" enums removed in https://github.com/smartdevicelink/sdl_ios/pull/1556/commits/9a0d9f6c8b0d2c553125880410dcf4db01f85115